ajax/libs/clone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "clone",
+  "filename": "clone.min.js",
+  "description": "deep cloning of objects and arrays",
+  "keywords": [
+    "clone",
+    "object",
+    "array",
+    "function",
+    "date"
+  ],
+  "version": "2.1.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pvorb/clone"
+  },
+  "author": "Paul Vorbach <paul@vorba.ch> (http://paul.vorba.ch/)",
+  "license": "MIT",
+  "npmName": "clone",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
@mercyg _`basePath`_ is the location that the auto-updater in cdnjs starts to fetch files. It will affect file structure and whether we could fetch correct files or not. 
Take this case as example, if you use `src` as basePath but there is no `src` directory in NPM package, auto-updater could not fetch the file. If we use `""` as basePath, it means at root of the package, we will see the files as following:
```
.
├── clone.iml
├── clone.js
├── LICENSE
├── node_modules
├── package.json
└── README.md
```

Then, auto-updater could start to fetch the files following the rules.
If somewhere is not clear, feel free to ask. Thanks :+1: